test/suite/TestOutputParser.test.ts

```diff
@@ -249,7 +249,27 @@ Test Case '-[MyTests.MyTests`,
                 darwinTestRegex
             );
             assert.strictEqual(runState.status, TestStatus.passed);
-            assert.strictEqual(runState.duration, 0.006);
+            assert.deepEqual(runState.timing, { duration: 0.006 });
+        });
+
+        test("Two single line failing tests", async () => {
+            const testRunState = new TestRunState(
+                ["MyTests.MyTests/testFailOne", "MyTests.MyTests/testFailTwo"],
+                true
+            );
+            const runState = testRunState.tests[0];
+            outputParser.parseResult(
+                `Test Case '-[MyTests.MyTests testFailOne]' started.
+/Users/user/Developer/MyTests/MyTests.swift:59: error: -[MyTests.MyTests testFailOne] : failed - One
+Test Case '-[MyTests.MyTests testFailOne]' failed (0.571 seconds).
+Test Case '-[MyTests.MyTests testFailTwo]' started.
+/Users/user/Developer/MyTests/MyTests.swift:61: error: -[MyTests.MyTests testFailTwo] : failed - Two
+Test Case '-[MyTests.MyTests testFailTwo]' failed (0.572 seconds).
+`,
```

Comment:
I meant to add a test for two failures within the same test

Reply:
Ah, sorry, my misunderstanding. I've now added a test for this case.